app/templates/components/forms/wizard/sessions-speakers-step.hbs

```diff
@@ -316,7 +316,7 @@
   {{else}}
     <div class="spacer-50"></div>
   {{/if}}
-
+  {{!--  Commeting until next release for thorough testing and better UI.
```

Comment:
Correct spelling of `Commenting`.

Reply:
@iamareebjamal @kushthedude  
I think we should change the comment to: 
`Commented out until issue #3598 is solved and needs thorough testing`.

Reply:
Hmm, sounds good.